codecs/text/src/test/java/com/datastax/oss/dsbulk/codecs/text/string/StringToIntegerCodecTest.java

```diff
@@ -27,22 +27,61 @@
 
 class StringToIntegerCodecTest {
 
-  private StringToIntegerCodec codec;
+  private StringToIntegerCodec codec1;
+  private StringToIntegerCodec codec2;
 
   @BeforeEach
-  void setUp() {
+  void setUpCodec1() {
```

Comment:
I realized that we don't have unit tests that test the default value of `codec.nullStrings` which is to leave the input untouched. I think it makes sense to add tests for two different cases:

1. Textual CQL types: validate that by default null stays null, and empty strings stay empty strings.
2. Non-textual CQL types: validate that null stays null, but empty strings should become nulls, as per the documentation.

 So I added a few tests in two classes: `StringToStringCodecTest` and `StringToIntegerCodecTest`.